client/components/LearnerViewDirective.js

```diff
@@ -67,6 +67,10 @@ tie.directive('learnerView', [function() {
                 <select class="tie-lang-select-menu" name="lang-select-menu">
                   <option value="Python" selected>Python</option>
                 </select>
+                <button class="tie-code-reset" name="code-reset"
+                    ng-click="resetCode()">
+                  Reset
```

Comment:
Change "Reset" to "Reset Code". This is to avoid confusion about what exactly the button is resetting.